sources/ade/CMakeLists.txt

```diff
@@ -47,12 +47,14 @@ if(BUILD_ADE_DOCUMENTATION)
                       VERBATIM)
 endif()
 
+include(GNUInstallDirs)
+
 install(TARGETS ade COMPONENT dev
         EXPORT adeTargets
-        ARCHIVE DESTINATION lib
-        LIBRARY DESTINATION lib
-        RUNTIME DESTINATION lib
-        INCLUDES DESTINATION include)
+	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}
+	INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 install(EXPORT adeTargets DESTINATION share/ade COMPONENT dev)
```

Comment:
Shouldn't the following install commands be changed too? E.g. `share` -> `${CMAKE_INSTALL_DATAROOTDIR}`.

Reply:
Also, please keep the file formating consistent.

Reply:
Done